sdk/src/main/java/com/kin/ecosystem/settings/presenter/SettingsPresenter.java

```diff
@@ -185,6 +187,10 @@ public void onFailure(Throwable throwable) {
 		});
 	}
 
+	private String getErrorMessage(Throwable throwable) {
+		return throwable.getCause() != null ? throwable.getCause().getMessage() : throwable.getMessage();
```

Comment:
could throwable be `null`?

Reply:
should not be in that case, but I'll check anyway.